classes/APIFAQClass.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+* 2007-2016 PrestaShop
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2015 PrestaShop SA
+* @license   http://addons.prestashop.com/en/content/12-terms-and-conditions-of-use
+* International Registered Trademark & Property of PrestaShop SA
+*/
+
+include_once(dirname(__FILE__).'/../../../config/config.inc.php');
```

Comment:
This line is not needed and can be removed. When your file is loaded, `config.inc.php` has been already included.

Reply:
fixed here : 6c386f937353985e9ccea2480200d366c9c7ffe9